dependencies/istio/install/README.md

```diff
@@ -8,9 +8,8 @@ We need custom configuration as discussed in the
 
 Specifically, the things we changed are:
 
-1. The policy of configmap `istio-sidecar-injector` is changed from `enabled` to `disabled`.
-   According to this [table](https://github.com/istio/istio/issues/6476#issuecomment-399219937),
-   `policy=disabled` and `namespace label=enabled` is what we want.
+1. The policy of configmap `istio-sidecar-injector` is `enabled`.
```

Comment:
So this is unchanged, should not be in this "list of changes".
But I want to keep the link to that table.
Can you move it to line 17?